src/constants/indicators.js

```diff
@@ -179,59 +179,89 @@ export default {
     contextual: false,
     source: 'Education attainment - Completed Year 12, % of population',
   },
+  educationalAttainmentCertificate: {
+    name: 'Educational attainment - Certificate',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Education attainment - Completed Certificate III or IV, % of population',
+  },
+  educationalAttainmentTertiary: {
+    name: 'Educational attainment - Tertiary',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
```

Comment:
Shouldn't all TODOs be gone in this task? Otherwise we need a new task to fix them.